manifests/plugin/foreman_gutterball.pp

```diff
@@ -1,12 +1,18 @@
 # foreman_gutterball plugin
 class katello_devel::plugin::foreman_gutterball {
   $path = "${katello_devel::deployment_dir}/foreman-gutterball"
+  $foreman_settings_path = "${katello_devel::deployment_dir}/foreman/config/settings.plugins.d"
```

Comment:
this is the correct dir for a devel install, not totally sure for prod install. still need to check. would it be possible to symlink the one from the foreman-gutterball checkout to here?

```sh
mkdir -p ~/foreman/config/settings.plugins.d/
ln -s ~/foreman-gutterball/config/foreman_gutterball.yaml ~/foreman/config/settings.plugins.d/
```